queue/call_outbound.go

```
@@ -2,6 +2,7 @@ package queue
 
 import (
 	"fmt"
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)